src/BOTIFY_PROJECT_AGGREGATE_URLS.js

```diff
@@ -164,5 +168,20 @@ function BOTIFY_PROJECT_AGGREGATE_URLS(
     }
   }
 
+  // Append errors
+  if (errors) {
+    var errors_list = [];
+    errors_list.length = result[0].length;
```

Comment:
To create an Array that have a X number of items, you can use this syntax: 

```
// This will create an array with 10 entries
var errors = new Array(10);
```

If you want to fill the array with default values you can use:

```
var a = new Array(10).fill(0);
```